### ISSUE TITLE
Research performance tests used by other frameworks

### ISSUE THREAD

**Issue by kubkon** [microsoft/fluentui#17217]
This is one of those Schroedinger's issue which might either come to a close really quickly, or unwrap into a rather lengthy, time-consuming research with many avenues. The TL;DR of this issue is to track and gather intel on performance tests used by other frameworks. As I will be working on analysing other frameworks and how it's done over there, I will periodically update this issue, so feel free to check back here for progress updates. Finally, whenever possible, I will try to port as many perf cases over to our own `perf-test` app as possible.

---
### tl;dr

- [ ] investigate performance scenarios used by other frameworks
    - [ ] Material -- wraps `React.Profiler` for performance metrics gather -- [link](https://github.com/mui-org/material-ui/blob/next/benchmark/browser/index.js)
- [ ] add additional performance scenarios to `perf-test` app
- [ ] draw some conclusions/compare against others to see how we fare at this moment

**Comment by miroslavstastny** [comment]
@kubkon - v0 runs perf tests on every build storing results to a database and rendering them in docsite:
![image](https://user-images.githubusercontent.com/9615899/110028901-21933900-7d34-11eb-9023-70eccaf5abbd.png)

It would be great if you would consider that as well.

**Comment by JustSlone** [comment]
Update:
- Working on getting example merged for flamegrill
- No updates otherwise

**Comment by JustSlone** [comment]
We'll resurrect if it's relevant later
